<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mentionlar</class>
 <widget class="QWidget" name="mentionlar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hashtagler</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Turkish" country="Turkey"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>84</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>84</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
